Methods.h

```diff
@@ -0,0 +1,50 @@
+#include "Classes.h"
+using namespace std;
+
+void Smartphone::add(float p, string n, float M, float A, float B)
+{
```

Comment:
You are using unclear names for the class methods, and it is unclear in what order the arguments are passed in the add method.

You should:
void Smartphone::add_InformationAboutSmartphone(flout smartphone_price_, string smartphone_model_name_, float the_amount_of_permanent_memory_by_gb_, float mobile_number, float battery_capacity_by_mah_) {

or

void Smartphone::add_InformationAboutSmartphone(flout price, string model_name, float permanent_memory, float mobile_number, float battery_capacity) {